docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,56 @@ tags:
 
 :::tip Support, stability, and dependency info
 
-- In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
-
-:::
+- API Keys used for Cloud Operations Tools (tcld, API, Terraform) are in [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API Keys used for Namespace Authentication are in [Pre Release](/evaluate/development-production-features/release-stages#pre-release)
+  :::
 
 Temporal Cloud supports secure programmatic access through API Key Authentication.
 
 ## Overview
 
-Temporal Cloud API Keys facilitate user-level authentication.
-Each key is unique to a single user and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+Temporal Cloud API Keys provide identity based authentication where an Identity can be a Temporal User or [Service Account](/cloud/service-accounts)
+Each key is unique to a single identity and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
 
 The process is as follows:
 
 ```text
-API Key (authentication) → User (identity) → RBAC (authorization)
+API Key (authentication) → Identity (User or Service Account) → RBAC (authorization)
 ```
 
-These API Keys are integral for authenticating Temporal Cloud operator tools, including [Temporal Cloud CLI (tcld)](/cloud/tcld) and the [Temporal Cloud Operations API](/ops).
-
-Users have the autonomy to create, delete, and update access to their API Keys through the Cloud UI or tcld.
-
-API Keys are an opt-in feature that must be enabled by a Global Administrator for use.
-For broader oversight, Global Administrators can manage user's API Keys using the same interfaces.
-For instructions on API Key Management for your organization, see [global Administrator API Key Management](#global-administrator-api-key-management).
-
-## Get started {#get-started}
-
-To get started, ensure you're a Temporal Cloud user.
-If you don't have an account with Temporal, **[sign up here](https://pages.temporal.io/get-started-with-cloud)**.
-
-### Prerequisites
-
-- A Cloud user account
-- Access to the [Temporal Cloud UI](https://cloud.temporal.io/) or Temporal Cloud CLI ([tcld](https://docs.temporal.io/cloud/tcld/))
-- [Enable access to API Keys](#manage-api-keys)
+API Keys support 2 main use cases:
 
-## Use API Keys {#use-api-keys}
+- _**Cloud Operations Automation**_: API Keys can be used as the authentication mechanism for Temporal Cloud operational tools ([Temporal Cloud CLI (tcld)](/cloud/tcld), [the Cloud Ops API](/ops), [Terraform](/production-deployment/cloud/terraform-provider)) to manage your Temporal Cloud account, namespaces, certificates, and user identities.
```

Comment:
It's simpler to say "Use API keys to authenticate with Temporal Cloud operational tools".

Please use casing consistently. For example, we have rules about Namespace being upper-cased. Applying the Vale linter will yelp. It is part of the docs repo.